lib/styles/components/dropdown.scss

```diff
@@ -1,5 +1,10 @@
 .bp3-popover.bp3-minimal.nui-dropdown--wrapper {
   box-shadow: none;
+
+  &.mt-2 {
+    margin-top: 0.5rem !important;
+  }
```

Comment:
This shouldn't be done. `mt-2` is from tailwind. @yedhink _a

Reply:
@vinay0x 
I copy pasted from `NitroHelp` stylesheets. I thought this was necessary since the change was made 1 month ago by Edwin.. 

Need confirmation to go ahead and remove this part from this PR.

Refer: https://github.com/bigbinary/neeto-help-web/blob/d7134ca3b2ce7185860f2d2f326e3e8775d04280/app/assets/stylesheets/v2/components/dropdown.scss#L3

Reply:
Remove it.